Gemfile

```
@@ -245,7 +245,8 @@ gem 'ansi'
 gem 'parallel'
 gem 'todo_or_die'
 
-gem 'business_time'
+gem 'dentaku', require: false
```

```suggestion
gem 'dentaku'
```

Whats the benefit of `require: false`? It wasn't installing for me by default so I was getting errors in the CE workflow, I had to install manually to get it working. Not sure how it will work when deployed.

---

Strange that is wasn't installing for you. The `require` arg just causes it not to be loaded by default, so you to `require` the gem explicitly.

This saves a little memory on installations that might never need the gem or use CE. But it's probably so marginal that it's not worth the hassle; it should be fine to remove the `require` directive.

---

Ah ok- there must have been something weird going on locally, its working for me now!